### DISCUSSION TITLE
Inquiry about the trajectory file

### DISCUSSION THREAD

**Discussion by kkang14** [materialsvirtuallab/m3gnet#46] [Q&A]
Dear developers,

Hope all goes well and thank you for a nice interatomic potential code.
I have an inquiry about the trajectory file.
I want to extract the force and energy in every step from the molecular dynamics.
But it seems the trajectory file is encoded.
How can I access the data?
Thank you!

Yours sincerely,
Kisung Kang

Postdoctoral researcher
Fritz-Haber-Institut der Max-Planck-Gesellschaft

**Comment by chc273** [comment]
Hi @kkang14  the trajectory file has the ASE format.https://wiki.fysik.dtu.dk/ase/ase/io/trajectory.html#examples
